acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/ibft2/BftAcceptanceTestParameterization.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.tests.acceptance.ibft2;
+
+import org.hyperledger.besu.tests.acceptance.dsl.AcceptanceTestBase;
+import org.hyperledger.besu.tests.acceptance.dsl.node.BesuNode;
+import org.hyperledger.besu.tests.acceptance.dsl.node.configuration.BesuNodeFactory;
+
+import java.util.List;
+
+import org.junit.Ignore;
+
+@Ignore("Must be ignored to prevent it being executed by JUnit")
+public class BftAcceptanceTestParameterization extends AcceptanceTestBase {
```

Comment:
Perhaps this class is in wrong place if we need an ignore. Perhaps move it into a support package or into the dsl?

Reply:
@Ignore has been removed.